src/Setup/Directory.Build.props

```diff
@@ -2,5 +2,8 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
   <PropertyGroup>
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
+    <!-- TODO: https://github.com/dotnet/source-build/issues/3981
+          Remove if necessary. -->
+    <ExcludeFromVerticalBuild>true</ExcludeFromVerticalBuild>
```

Comment:
Having both properties seems unnecessary. What code is excuded from the vertical build but included in source build?

Reply:
Expect this to go away when the new Unified build controls come to this repo. That will happen after the implementation is complete (very soon) and when Roslyn moves to .NET 9 Arcade (late). Whenever you see both of these conditions, they would be replaced by `ExcludeFromDotNetBuild`